common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -515,6 +521,10 @@ public void onReceive(Context context, Intent intent) {
                         // set correlation id on Local Authentication Result
                         setCorrelationIdOnResult(commandResult, correlationId);
 
+                        Logger.info(TAG + methodName, "",
```

Comment:
It just seems confusing to be passing around these empty strings for correlation ids when we do have one actually

Reply:
Noted, will refactor it, again this was done in accordance with how the logging is and was(partly) strucutred.

Reply:
Made the change https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1237/commits/e3620b61cfa0376f89d61d646bfe46340d2f295d